wallet/src/de/schildbach/wallet/ui/BuyAndSellLiquidUpholdActivity.kt

```diff
@@ -25,26 +25,31 @@ import android.view.MenuItem
 import android.view.View
 import androidx.appcompat.widget.Toolbar
 import androidx.core.os.bundleOf
+import androidx.core.view.isGone
 import androidx.core.view.isVisible
 import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProvider
 import com.google.gson.Gson
 import com.google.gson.reflect.TypeToken
+import dagger.hilt.android.AndroidEntryPoint
 import de.schildbach.wallet.Constants
 import de.schildbach.wallet.WalletApplication
 import de.schildbach.wallet.dialog.CurrencyDialog
 import de.schildbach.wallet.rates.ExchangeRatesViewModel
+import de.schildbach.wallet.ui.coinbase.CoinBaseWebClientActivity
+import de.schildbach.wallet.ui.coinbase.CoinbaseActivity
 import de.schildbach.wallet_test.R
 import kotlinx.android.synthetic.main.activity_buy_and_sell_liquid_uphold.*
+import kotlinx.android.synthetic.main.item_coinbase_integration_item.*
```

Comment:
Maybe we should get rid of kotlin synthetics while we're at it

Reply:
Ideally new fragments/activities should not use synthetics.  `BuyAndSellLiquidUpholdActivity` is not new and may still use synthetics.